### ISSUE TITLE
bug(mac): Generate error when Keyman version > Keyman for macOS version

### ISSUE THREAD

**Issue by darcywong00** [keymanapp/keyman#11053]
I was troubleshooting with a Keyman for macOS user for keymanapp/keyboards#2624

On his Intel MacBook Pro 16 Sonoma 14.4, the kmp file failed to install

> Nothing happens when I double click on the file. When I drag it nothing happens. Or rather it “undrags” itself back to the download folder

![keyman macos fail](https://github.com/keymanapp/keyman/assets/7358010/6d53025e-5410-4758-a0b6-caf1a2e207a7)

We checked the [macOS security permissions](https://help.keyman.com/products/mac/current-version/troubleshooting/configure-security) and they were fine.
When we tried installing the kmp file on a PC, Keyman for Windows generated a useful reason

![Keyman version warning](https://github.com/keymanapp/keyman/assets/7358010/cd313305-c18f-462d-aee9-1dfe7dcffc40)

It would be nice if Keyman for macOS could generate a similar warning if that's the reason a kmp file can't be installed.

imho, low priority for post-17.0 release

**Comment by mcdurdin** [comment]
Yep, agree triage to 18.0. Question: does the keyboard in question need to be 17.0? Is it using any 17.0-specific features?

**Comment by darcywong00** [comment]
> Yep, agree triage to 18.0. Question: does the keyboard in question need to be 17.0? Is it using any 17.0-specific features?

I was originally thinking of adding flick gestures, but turns out not needed.

**Comment by mcdurdin** [comment]
Relates to https://github.com/keymanapp/keyman/pull/15635#discussion_r3304139051
